internal/e2e_test.go

```diff
@@ -0,0 +1,132 @@
+//go:build e2e
+
+package internal
+
+import (
+	"context"
+	"fmt"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/corazawaf/coraza/v3/http/e2e"
+	"github.com/mccutchen/go-httpbin/v2/httpbin"
+	"github.com/rs/zerolog"
+
+	"github.com/dropmorepackets/haproxy-go/pkg/testutil"
+)
+
+const directives = `
```

Comment:
Please use `go:embed` for this, also in general we would like to keep a consistent http e2e suite by suing https://github.com/corazawaf/coraza/tree/main/http/e2e which is being used in coraza http handler, caddy and coraza-proxy-wasm. Running it is as easy as spin up an instance and run a go command, see https://github.com/corazawaf/coraza-caddy/blob/main/magefile.go#L74

Reply:
Why should I use embed for a single string? Inside the coraza tests its done the same way.

Reply:
But yeah I can migrate the e2e tests to run the command but I am not a fan of running commands and basically ignoring the go test framework. If the Upstream would properly expose the tests as a list we could run these in parallel and reduce testing times while also reducing the amount of external dependencies

Reply:
I opened an issue on coraza itself to expose this and make it easier for other repos to use the e2e test suite.
https://github.com/corazawaf/coraza/issues/1006

Reply:
Waiting for a coraza release to fix this up

Reply:
Maybe it is better you use master for now.